src/main/java/ulcrs/controllers/CourseController.java

```diff
@@ -30,12 +30,14 @@ public RouteGroup routes() {
 
     private List<Course> getCourseList(Request request, Response response) {
         response.type(CONTENT_TYPE_JSON);
-        return DataStore.getCourses();
+        String cookie = request.headers("Set-Cookie");

```

Comment:
Is it set-cookie or is there a just cookie field? Sounds like set-cookie will be processed to real cookie when sending the API request

Reply:
This is the only possible change I can think of for now. If it's okay then the PR is good.

Reply:
this is how it works now, so for the sake of time, I've left it. we can look into it in the future. the cookies() method does not work by itself.